Project-Hotpot/Project-Hotpot/APIManager.swift

```diff
@@ -0,0 +1,209 @@
+//
+//  APIManager.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/11/22.
+//
+
+import Foundation
+import UIKit
+
+class APIManager: NSObject {
```

Comment:
Nit: Consider having more explicit naming like SpotifyManager. API sounds generic and could mean this class handles all kinds of APIs like soundcloud etc.